easybuild/easyconfigs/e/EpiSCORE/EpiSCORE-0.9.5-20220621-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'RPackage'
+
+name = 'EpiSCORE'
+local_commit = '4f7daef'
```

Comment:
I don't think this isn't guaranteed to be unique, better to use full commit string.

Reply:
We use the short commit in every single easyconfig that uses local_commit, so i don't think we should start enforcing it here on a repo with 12 commits (with 7 characters still gives 78e9 combinations which assuming a repo has a thousand commits would mean a 1 in which even at 7 character is still 78e9 combinations